index.js

```diff
@@ -62,72 +29,56 @@ function prepareLogs(eventData, log_raw_event) {
     , meta: {
         owner: eventData.owner
       , filters: eventData.subscriptionFilters
-      }, line: JSON.stringify({message: event.message, ...eventMetadata})
+      }
+    , line: event.message
     }
 
-    if (log_raw_event) {
-      eventLog.line = event.message
-      eventLog.meta = {...eventLog.meta, ...eventMetadata}
-    }
+    eventLog.meta = {...eventLog.meta, ...eventMetadata}
```

Comment:
Instead of overwriting the `eventLog.meta` with data from itself and `eventMetaData`, we can just spread the `eventMetadata` during the eventLog creation